arangodb-net-standard/AqlFunctionApi/AqlFunctionApiClient.cs

```diff
@@ -507,5 +508,30 @@ public AqlFunctionApiClient(IApiClientTransport transport, IApiClientSerializati
                 throw await GetApiErrorException(response).ConfigureAwait(false);
             }
         }
+
+        /// <summary>
+        /// Gets the available optimizer rules for AQL queries
+        /// GET /_api/query/rules
+        /// </summary>
+        /// <param name="token">A CancellationToken to observe while waiting for the task to complete or to cancel the task.</param>
+        /// <remarks>
+        /// Returns an array of objects that contain the name of each available 
+        /// rule and its respective flags.
+        /// </remarks>
+        /// <returns></returns>
+        public virtual async Task<List<GetQueryRulesResponseItem>> GetQueryRulesAsync(CancellationToken token = default)
```

Comment:
Can you add a simple unit test to this method? To verify that the method is working and everything is deserialized fine.

Reply:
Good point. I've added the test.

Reply:
GetQueryRulesAsync_ShouldSucceed